TemperatureControl/TemperatureControl.js

```diff
@@ -2,12 +2,7 @@
  * Provides Agate-themed temperature control components and behaviors.
  *
  * @example
- * <TemperatureControl
- *   max={30}
- *   min={10}
- *   unit="Celsius"
- *   value={10}
- * />
+ * <TemperatureControl min={0} max={40} unit="Fahrenheit" />
```

Comment:
When I check the example, is shown by Celsius. I don't know the reason. Did i something wrong or is there issue on the component?